src/include/distributed/commit_protocol.h

```diff
@@ -14,18 +14,12 @@
 
 
 #include "access/xact.h"
+#include "distributed/connection_management.h"
 #include "libpq-fe.h"
 #include "lib/stringinfo.h"
 #include "nodes/pg_list.h"
 
 
-/* Enumeration that defines the different commit protocols available */
-typedef enum
-{
-	COMMIT_PROTOCOL_1PC = 0,
-	COMMIT_PROTOCOL_2PC = 1
-} CommitProtocolType;
```

Comment:
We can delay it, but for me it seems a lot more appropriate to have that in transaction_management.h. You want to push that back?